### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by g-bottino**
this is my commit

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Hello
 
 text
+text #2
```
Comment (StarLederer) [inline index.md:4]: This is not elaborate enough

File: index.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
 # Hello
```
Comment (StarLederer) [inline index.md:1]: I was hoping this would changed too

**Review by StarLederer — CHANGES_REQUESTED** [review]
Could you please make the request changes

**Comment by g-bottino** [comment]
dispute this. Let me fix it myself.

File: index.md (hunk 2 above)
Comment (g-bottino) [reply index.md:1]: here is where the comment was supposed to be

File: index.md [hunk 3]
```diff
@@ -1,3 +1,4 @@
...
 
 text
+text #2 whgich is ellaborate
```
Comment (StarLederer) [inline index.md:4]: thank

**Review by StarLederer — APPROVED** [review]

**Pull request merged**